### DISCUSSION TITLE
Newb can't navigate to next error

### DISCUSSION THREAD

**Discussion by danbills** [mfussenegger/nvim-jdtls#394] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/165320/208711065-eadd8306-ede5-451a-b0a6-986bc63f3d94.png)

Hello, I'm trying to navigate to errors using `:cnext` and `:lnext` to no avail.

`:cnext` shows `E42: No errors` and `:lnext` says `E776: No location list`.

I'm sure I'm missing something basic, but I think I'm at the end of my vim-fu and am stuck.

**Comment by mfussenegger** [comment, marked as answer]
The language server client in neovim reports diagnostics via the `vim.diagnostic` module. See `:help vim.diagnostic`.

You can move to the next/previous using either `:lua vim.diagnostic.goto_next()` or `:lua vim.diagnostic.goto_prev()`
